modules/apps/transfer/types/keys.go

```diff
@@ -87,5 +85,5 @@ func TotalEscrowForDenomKey(denom string) []byte {
 // PacketForwardKey returns the store key under which the forwarded packet is stored
 // for the provided portID, channelID, and packet sequence.
 func PacketForwardKey(portID, channelID string, sequence uint64) []byte {
-	return []byte(fmt.Sprintf("%s/%s/%s/%d", KeyPacketForwardPrefix, portID, channelID, sequence))
+	return []byte(fmt.Sprintf("%s/%s/%s/%d", forwardPacketKey, portID, channelID, sdk.Uint64ToBigEndian(sequence)))
```

Comment:
This is decimal encoding for byte slice, is this correct / what we want here?

Was this something for iteration or something? We don't do this `Uint64ToBigEndian` conversion in the regular packet keys in 24-host.